README.md

```diff
@@ -137,6 +142,99 @@ The above would ensure that you run Pa11y CI against local URLs instead of the l
 
 If there are items in the sitemap that you'd like to exclude from the testing (for example PDFs) you can do so using the `--sitemap-exclude` flag.
 
+## Reporters
+
+Pa11y CI supports Pa11y compatible reporters. You can use the `--reporter` option to define a single reporter. The option value can be:
+- the path of a locally installed npm module (ie: `pa11y-reporter-html`)
+- the path to a local node module relative to the current working directory (ie: `./reporters/my-reporter.js`)
+- an absolute path to a node module (ie: `/root/user/me/reporters/my-reporter.js`)
+
+Example:
+
+```
+npm install pa11y-reporter-html --save
+pa11y-ci --reporter=pa11y-reporter-html http://pa11y.org/
+```
+
+**Note**: When using reporters that output to stdout, all pa11y-ci execution logs will be redirected to stderr. This allows you to
```

Comment:
I don't think we should write to `stderr` with non-error output. `stderr` does mean something, and output to it can be considered a failure.
IMO `pa11y-ci` execution logs should still be outputted to `stdout`.

Reply:
I remembered reading this solution on the jest source code (https://github.com/facebook/jest/blob/420bcb7115bc40429ba4737ede0d3d1dd72a472f/packages/jest-core/src/cli/index.ts#L48) to prevent mixing CLI output to reporter output.

If there's a better (and indeed less ambiguous) option I'd be happy to make the change.

Also preventing normal pa11y when using reporters might be a solution as well.

Reply:
> Wouldn't the json output still be broken as it's multiple JSON objects?
Yes, using pa11y-reporter-json the output will be anyway broken. Custom reporters that can output a header and a footer using `beforeAll` and `afterAll` would not have this issue.

My idea was to re-use pa11y reporters as well, but I agree their output could be broken or incomplete.  Maybe we could add a check for a compatibility flag  (like `module.exports.supportPa11yCI`) and warn the user when the reporter is not fully compatible?

> Either way, I think this is a good addition but only if the reporter is outputting json

Do you mean the `json` flag or when the reporter name matches `/.+json.+/`?